cloudbeesjenkins-dockerdatacenter/README.md

```diff
@@ -7,34 +7,37 @@
 </a>
 https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/cloudbeesjenkins-dockerdatacenter
 ## Azure Partner Quickstarts Overview
-***Azure Partner Quickstart Templates*** provide customers with a highly automated process to launch enterprise ready first and 3rd party ISV solution stacks on Azure in a pre-production environment. The **Azure Partner Quickstart** effort is complimentary to the [Azure Marketplace test drive program](https://azure.microsoft.com/en-us/marketplace/test-drives/). These fully baked stacks enable customers to quickly stand up a PoC or Piloting environments and also integrate it with their systems and customization.
+***Azure Partner Quickstart Templates*** provide customers with a highly automated process to launch enterprise ready 1st and 3rd party ISV solution stacks on Azure in a pre-production environment. The **Azure Partner Quickstart** effort is complimentary to the [Azure Marketplace test drive program](https://azure.microsoft.com/en-us/blog/azure-marketplace-test-drive/). These fully baked stacks enable customers to quickly stand up a Proof of Concept or Pilot environments, as well as integrate it with their existing systems and configurations.
```

Comment:
Suggest some edits:

[Azure Marketplace Test Drive Program](https://azure.microsoft.com/en-us/blog/azure-marketplace-test-drive). These fully baked stacks enable customers to quickly stand up a Proof of Concept or Pilot environment, as well as integrate it with their existing systems and configurations.